src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
```

Comment:
header comment is really important..  should be done such that javadoc will pick it up.